monitoring/mongodb/dashboard.json

```
@@ -265,1465 +341,5690 @@
+                "value": "negative-Y"
+              }
+            ]
+          }
+        ]
+      },
+      "gridPos": {
+        "h": 8,
+        "w": 12,
+        "x": 0,
+        "y": 132
       },
-      "type": "graph",
-      "xaxis": {
-        "buckets": null,
-        "mode": "time",
-        "name": null,
-        "show": true,
-        "values": []
+      "id": 1041,
+      "options": {
+        "legend": {
+          "calcs": [
+            "mean",
+            "max",
+            "min"
+          ],
+          "displayMode": "table",
+          "placement": "bottom",
+          "showLegend": true
+        },
+        "tooltip": {
+          "mode": "multi",
+          "sort": "none"
+        }
       },
-      "yaxes": [
+      "pluginVersion": "9.2.20",
+      "targets": [
         {
-          "format": "binBps",
-          "label": "",
-          "logBase": 1,
-          "max": null,
-          "min": null,
-          "show": true
+          "datasource": {
+            "type": "prometheus",
+            "uid": "metalk8s-prometheus"
+          },
+          "editorMode": "code",
+          "expr": "avg by (type) ((max_over_time(mongodb_mongod_wiredtiger_concurrent_transactions_total_tickets{txn_rw=\"read\"}[$interval]) or\nmax_over_time(mongodb_mongod_wiredtiger_concurrent_transactions_total_tickets{txn_rw=\"read\"}[5m])) -\n(max_over_time(mongodb_mongod_wiredtiger_concurrent_transactions_out_tickets{txn_rw=\"read\"}[$interval]) or\nmax_over_time(mongodb_mongod_wiredtiger_concurrent_transactions_out_tickets{txn_rw=\"read\"}[5m])))",
+          "interval": "$interval",
+          "intervalFactor": 1,
+          "legendFormat": "read",
+          "metric": "",
+          "range": true,
+          "refId": "B",
+          "step": 300
         },
         {
-          "format": "short",
-          "label": null,
-          "logBase": 1,
-          "max": null,
-          "min": null,
-          "show": true
+          "datasource": {
+            "type": "prometheus",
+            "uid": "metalk8s-prometheus"
+          },
+          "editorMode": "code",
+          "expr": "avg by (type) ((max_over_time(mongodb_mongod_wiredtiger_concurrent_transactions_total_tickets{txn_rw=\"write\"}[$interval]) or\nmax_over_time(mongodb_mongod_wiredtiger_concurrent_transactions_total_tickets{txn_rw=\"write\"}[5m])) -\n(max_over_time(mongodb_mongod_wiredtiger_concurrent_transactions_out_tickets{txn_rw=\"write\"}[$interval]) or\nmax_over_time(mongodb_mongod_wiredtiger_concurrent_transactions_out_tickets{txn_rw=\"write\"}[5m])))",
+          "hide": false,
+          "interval": "$interval",
+          "intervalFactor": 1,
+          "legendFormat": "write",
+          "metric": "",
+          "range": true,
+          "refId": "A",
+          "step": 300
         }
       ],
-      "yaxis": {
-        "align": false,
-        "alignLevel": null
-      }
+      "title": "WiredTiger Concurrency Tickets Available",
```

Wiretiger is redundant here, this is the title of the section
```suggestion
      "title": "Available Concurrency Tickets",
```